eZ/Publish/Core/Persistence/Cache/UserHandler.php

```diff
@@ -24,6 +24,8 @@ class UserHandler extends AbstractHandler implements UserHandlerInterface
 {
     /**
      * {@inheritdoc}
+     *
+     * @throws \Psr\Cache\InvalidArgumentException
```

Comment:
invalid edits, not for this PR, but if you are truly seeing this exception then we need to catch it inside the cache impl here and not let it leak true.

Reply:
I never saw them per se, that was just catch up by static code analysis. Should I then revert those?